src/main/resources/promote-maven-plugin.properties

```diff
@@ -0,0 +1,3 @@
+groupId=${project.groupId}
+artifactId=${project.artifactId}
+version=${project.version}
```

Comment:
The irony here is that this means the promot plugin cannot be used to release itself :-)

Reply:
Ha...I hadn't thought about it. Perhaps we need to make it configurable?